### PR TITLE
138 cc translation

### LINKED ISSUE #138: Add CC Rework translation

### PR DESCRIPTION
**Pull request by clementcharmillot**
Adds CC translation for the builder.

Related to:
- graasp/graasp-builder/pull/596
- graasp/graasp-library/pull/234

closes #138

### CODE REVIEW

File: src/langs/en/library.json [hunk 1]
```diff
@@ -74,5 +74,24 @@
...
+  "SUMMARY_DETAILS_LICENSE_TITLE": "License",
+  "SUMMARY_BROWSE_FILE_UPDATED": "Updated {{date}}",
+  "SUMMARY_BROWSE_FOLDER_CONTAINS_zero": "Empty Folder",
+  "SUMMARY_BROWSE_FOLDER_CONTAINS_one": "{{count}} File",
+  "SUMMARY_BROWSE_FOLDER_CONTAINS_other": "{{count}} Files",
+  "SUMMARY_DESCRIPTION_SHOW_MORE": "Show More",
+  "SUMMARY_DESCRIPTION_SHOW_LESS": "Show Less",
+  "SUMMARY_ITEMS_SHOW_LESS_one": "- Hide {{count}} item",
+  "SUMMARY_ITEMS_SHOW_LESS_other": "- Hide {{count}} items",
+  "SUMMARY_ITEMS_SHOW_MORE": "+ View {{count}} more",
```
Comment (spaenleh) [inline src/langs/en/library.json:92]: As said in the library review, I think you should remove these `-` and `+` and add them as icons on the button.

File: src/langs/en/library.json [hunk 2]
```diff
@@ -74,5 +74,24 @@
...
+  "SUMMARY_ACTIONS_PREVIEW_CONTENT": "Preview Content",
```
Comment (spaenleh) [inline src/langs/en/library.json:93]: Maybe simply use 
```suggestion
  "SUMMARY_ACTIONS_PREVIEW_CONTENT": "Preview",
```
as you already have the "play" button.

File: src/langs/fr/library.json [hunk 3]
```diff
@@ -29,5 +29,25 @@
...
+  "SUMMARY_BROWSE_FILE_UPDATED": "Mis à jour {{date}}",
+  "SUMMARY_BROWSE_FOLDER_CONTAINS_zero": "Dossier vide",
+  "SUMMARY_BROWSE_FOLDER_CONTAINS_one": "{{count}} fichier",
+  "SUMMARY_BROWSE_FOLDER_CONTAINS_other": "{{count}} fichiers",
+  "SUMMARY_DESCRIPTION_SHOW_MORE": "Montrer Plus",
+  "SUMMARY_DESCRIPTION_SHOW_LESS": "Montrer Moins",
+  "SUMMARY_ITEMS_SHOW_LESS_one": "- Cacher {{count}} élément",
+  "SUMMARY_ITEMS_SHOW_LESS_other": "- Cacher {{count}} éléments",
+  "SUMMARY_ITEMS_SHOW_MORE": "+ Montrer {{count}} éléments",
+  "SUMMARY_ACTIONS_PREVIEW_CONTENT": "Aperçu du contenu",
```
Comment (spaenleh) [inline src/langs/fr/library.json:49]: Same here
```suggestion
  "SUMMARY_ACTIONS_PREVIEW_CONTENT": "Aperçu",
```

File: src/langs/fr/library.json [hunk 4]
```diff
@@ -29,5 +29,25 @@
...
+  "SUMMARY_DETAILS_LICENSE_TITLE": "Licence",
+  "SUMMARY_BROWSE_FILE_UPDATED": "Mis à jour {{date}}",
+  "SUMMARY_BROWSE_FOLDER_CONTAINS_zero": "Dossier vide",
+  "SUMMARY_BROWSE_FOLDER_CONTAINS_one": "{{count}} fichier",
+  "SUMMARY_BROWSE_FOLDER_CONTAINS_other": "{{count}} fichiers",
+  "SUMMARY_DESCRIPTION_SHOW_MORE": "Montrer Plus",
+  "SUMMARY_DESCRIPTION_SHOW_LESS": "Montrer Moins",
+  "SUMMARY_ITEMS_SHOW_LESS_one": "- Cacher {{count}} élément",
+  "SUMMARY_ITEMS_SHOW_LESS_other": "- Cacher {{count}} éléments",
+  "SUMMARY_ITEMS_SHOW_MORE": "+ Montrer {{count}} éléments",
```
Comment (spaenleh) [inline src/langs/fr/library.json:48]: Same remove `-` and `+`

File: src/langs/fr/library.json [hunk 5]
```diff
@@ -29,5 +29,25 @@
   "SEARCH_RANGES_TITLE": "Titre",
   "SEARCH_RANGES_TAGS": "Tag",
   "SHARED_ITEMS_LABEL": "Éléments Partagés",
-  "OWN_ITEMS_LABEL": "Mes Éléments"
+  "OWN_ITEMS_LABEL": "Mes Éléments",
+  "COLLECTION_EMPTY_DESCRIPTION_TEXT": "Cet élément n'a pas de description.",
+  "SUMMARY_DETAILS_TITLE": "Détails",
+  "SUMMARY_DETAILS_CREATED_AT_TITLE": "Créé",
+  "SUMMARY_DETAILS_UPDATED_AT_TITLE": "Mis à jour",
+  "SUMMARY_DETAILS_NO_LANGUAGES": "Cet élément n'a pas de language défini.",
```
Comment (spaenleh) [inline src/langs/fr/library.json:37]:
```suggestion
  "SUMMARY_DETAILS_NO_LANGUAGES": "Cet élément n'a pas de langue définie.",
```

**Review by spaenleh — CHANGES_REQUESTED** [review]
⚡ Please have a look at my comments, happy to discuss.

**Review by dialexo — APPROVED** [review]
LGTM :heavy_check_mark: consider @spaenleh 's comments, otherwise we can just merge :+1:

**Review by spaenleh — APPROVED** [review]

**Review by spaenleh — APPROVED** [review]
LGTM ! 🚀

**Pull request merged**